backend/internal/environment/environment.go

```diff
@@ -32,6 +32,18 @@ const (
 	// Environment variable used to configure the allowed origins for
 	// accessing the API.
 	EnvVarAPIAllowedOrigins = "ZEEVISION_API_ALLOWED_ORIGINS"
+	// Environment variable used to configure the database username
+	EnvVarDatabaseUser = "ZEEVISION_DATABASE_USER" //nolint:all
+	// Environment variable used to configure the database user password
+	EnvVarDatabasePassword = "ZEEVISION_DATABASE_PASSWORD" //nolint:all
+	// Environment variable used to configure the database name
+	EnvVarDatabaseName = "ZEEVISION_DATABASE_NAME"
+	// Environment variable used to configure if the database should be
+	// hosted.
+	EnvVarHostDatabase = "ZEEVISION_HOST_DATABASE"
+	// Environment variable used to configure the port to use for the
+	// database.
+	EnvVarDatabasePort = "ZEEVISION_DATABASE_PORT"
```

Comment:
I think the comment here should something like that this port is used to connect to the database, not for hosting it like it currently sounds like.